src/main/java/cceclipseplugin/core/PluginManager.java

```diff
@@ -387,12 +385,12 @@ private void registerNotificationHooks() {
 				return;
 			}
 			Project project = dataManager.getSessionStorage().getProjectById(mm.getProjectIDForFileID(resId));
-			String pathToFile = meta.getFilePath();
 			IProject p = ResourcesPlugin.getWorkspace().getRoot().getProject(project.getName());
-			IFile file = p.getFile(pathToFile);
+			IFile file = p.getFile(meta.getFilePath());
+			String fullPath = file.getFullPath().toString();
```

Comment:
Refactor this variable to specify what type of path this is (workspace relative, project relative with or without file name, or absolute)